core/embed/rust/src/trezorhal/display.rs

```diff
@@ -0,0 +1,69 @@
+extern "C" {
+    // trezorhal/display.c
+    fn display_backlight(val: cty::c_int) -> cty::c_int;
+    fn display_text(
+        x: cty::c_int,
+        y: cty::c_int,
+        text: *const cty::c_char,
+        textlen: cty::c_int,
+        font: cty::c_int,
+        fgcolor: cty::uint16_t,
+        bgcolor: cty::uint16_t,
+    );
+    fn display_text_width(
+        text: *const cty::c_char,
+        textlen: cty::c_int,
+        font: cty::c_int,
+    ) -> cty::c_int;
+    fn display_bar(x: cty::c_int, y: cty::c_int, w: cty::c_int, h: cty::c_int, c: cty::uint16_t);
+    fn display_bar_radius(
+        x: cty::c_int,
+        y: cty::c_int,
+        w: cty::c_int,
+        h: cty::c_int,
+        c: cty::uint16_t,
+        b: cty::uint16_t,
+        r: cty::uint8_t,
+    );
+}
+
+const WIDTH: i32 = 240;
+const HEIGHT: i32 = 240;
+
+pub fn width() -> i32 {
```

Comment:
why do these functions exist? as opposed to accessing the const directly

Reply:
I like it 🤷‍♂️